app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PagesController < ApplicationController
+    def show
+        render template: "pages/#{params[:page]}"
+    end
+end
```

Comment:
use 2 spaces for code alignment

Reply:
Noted Maam, will include in the next pull request